File: library/src/main/java/com/dhy/debugutil/data/ConfigRequest.java
package com.dhy.debugutil.data;

import java.io.Serializable;

public class ConfigRequest implements Serializable {
    private String applicationId;
    private String token;
    private String sqlId;
    private Object data;

    public <DATA extends Serializable> ConfigRequest(String applicationId, String token, String sqlId, DATA data) {
        this.applicationId = applicationId;
        this.token = token;
        this.sqlId = sqlId;
        this.data = data;
    }
}

File: library/src/main/java/com/dhy/debugutil/data/ConfigRequest.kt
package com.dhy.debugutil.data

import com.google.gson.Gson
import java.io.Serializable

class ConfigRequest(appId: String, configName: String) : Serializable {
    private var applicationId: String = appId
    private var name: String = configName
    var data: List<RemoteConfig>? = null
    override fun toString(): String {
        return Gson().toJson(this)
    }
}
